DDCore/CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ ENDIF()
 
 # install python files
 file(GLOB DDCore_Python python/*.py)
-install(FILES ${DDCore_Python} DESTINATION python)
+install(FILES ${DDCore_Python} DESTINATION lib/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages)
```

Comment:
Create a  variable like DD4HEP_PYTHON_INSTALL_DIR and use that throughout.

Reply:
Done